# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium-shim/patches/with_update/patch_order.txt
# Commit: 7ba557fab26e
# Full Hash: 7ba557fab26e27e366372eaf3d78997f71185b20
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-07-25 21:49:06
# Description:
#   Bug 1565848: Revert latest change to MITIGATION_DLL_SEARCH_ORDER. r=aklotz
#   
#   This is until any regressions can be fixed, see bug 1568850.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D39357
# ==============================================================================

diff -r d9ac2124b930 -r 7ba557fab26e security/sandbox/chromium-shim/patches/with_update/patch_order.txt
--- a/security/sandbox/chromium-shim/patches/with_update/patch_order.txt	Thu Jul 25 21:26:27 2019 +0300
+++ b/security/sandbox/chromium-shim/patches/with_update/patch_order.txt	Thu Jul 25 17:44:24 2019 +0000
@@ -23,3 +23,4 @@
 add_return_in_QueryCancellationTraitsForNonCancellables_to_satisfy_build.patch
 include_atomic_header_in_platform_thread.patch
 aarch64_control_flow_guard.patch
+revert_removal_of_app_dir_for_DLL_load.patch